packages/compile-solidity/src/compilerSupplier/loadingStrategies/VersionRange.ts

```diff
@@ -6,7 +6,19 @@ import originalRequire from "original-require";
 
 // must polyfill AbortController to use axios >=0.20.0, <=0.27.2 on node <= v14.x
 import "../../polyfill";
-import { default as axios, AxiosResponse } from "axios";
+import { default as axios_plain, AxiosResponse } from "axios";
+
+var proxy = process.env.https_proxy || process.env.http_proxy;
```

Comment:
This should be a `const`, preferably a more descriptive name such as `environmentProxy`